modules/apps/frontend-js/frontend-js-react-web/tsconfig.json

```diff
@@ -28,7 +28,8 @@
 	},
 	"include": [
 		"src/**/*",
-		"test/**/*"
+		"test/**/*",
+		"../../../global-env.d.ts"
```

Comment:
I'd add this change as its own commit and label it "TEMP". I will be updating portal with npm-scripts to provide this and we won't need to manually add this change.

Reply:
Yea, I read that but forgot about it when writing the PR description 👍